go.sum

```diff
@@ -1,144 +1,671 @@
+4d63.com/gochecknoglobals v0.0.0-20201008074935-acfc0b28355a h1:wFEQiK85fRsEVF0CRrPAos5LoAryUsIX1kPW/WrIqFw=
```

Comment:
Will you run `make tidy` and repush this file? There seems to be many added dependencies that shouldn't necessarily be here.

Reply:
[`2b4f8d4` (#47)](https://github.com/open-telemetry/opentelemetry-log-collection/pull/47/commits/2b4f8d4b2096106383b9105bbbdc6fa635531578) should fix it, let me reset the change ant re-fix it

Reply:
okay, i think this problem is fixed